assets/js/ally.js

```diff
@@ -174,24 +174,26 @@
                 }
 
                 if (isValid) {
-                    var data = $(this).serialize();
                     if (!selectedCountryHasRequiredStateField()) {
-                        data = data.replace('StateorProvince=', '');
+                        formData.StateorProvince = '';
                     }
                     if (!$('#form-optin').prop('checked')) {
-                        data = data + '&OptIn=0';
+                        formData.OptIn = 0;
                     }
                     $.ajax({
-                        'url': 'https://s2376.t.eloqua.com/e/f2',
+                        'url': 'https://2m0qbgl101.execute-api.us-east-1.amazonaws.com/dev',
```

Comment:
Which AWS account is this running under? Is it ally-prod?

Reply:
It's ally-dev. Should I change it over to prod?